### PR TITLE
Update maplibre to 4.7.1

### PR DESCRIPTION
**Pull request by aapeliv**
**Web frontend checklist**
- [ ] Formatted my code with `yarn format`
- [ ] There are no warnings from `yarn lint --fix`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/web/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "grpc-web": "1.5.0",
     "intersection-observer": "^0.12.2",
     "luhn": "^2.4.1",
-    "maplibre-gl": "^1.15.3",
+    "maplibre-gl": "^4.7.1",
```
Comment (bakeiro) [inline app/web/package.json:45]: such a big change...

**Comment by bakeiro** [comment]
I can test this, I can look for errors/warnings/different behaviors.... then I approve it?

**Comment by aapeliv** [comment]
Yes please! Thank you :)

**Comment by bakeiro** [comment]
I'm getting an error (image attached)

image


I guess you can do this to avoid it
```
if (map.isLoaded()) {...} // or something similar
```

**Comment by bakeiro** [comment]
another one after clicking "search here" button...
image

**Comment by bakeiro** [comment]
same when I tried to apply some filters, that broke the page

**Comment by nabramow** [comment]
You probably already know this but here's the list of breaking changes for each version upgrade. I usually upgrade one version at a time so it's only one set of breaking changes at a time.

v2: https://github.com/maplibre/maplibre-gl-js/blob/main/CHANGELOG.md#features-and-improvements-2
v3: https://github.com/maplibre/maplibre-gl-js/blob/main/CHANGELOG.md#potentially-breaking-changes
v4: https://github.com/maplibre/maplibre-gl-js/blob/main/CHANGELOG.md#400

Then at least you know what to look for...

**Comment by nabramow** [comment]
Closing as I'm taking this over and will do it one major version at a time starting with #5105

**Pull request closed without merging**